quartz/i18n/locales/de-DE.ts

```diff
@@ -1,6 +1,21 @@
 import { Translation } from "./definition"
 
 export default {
+  callout: {
+    note: "Hinweis",
+    abstract: "Zusammenfassung",
+    info: "Info",
+    todo: "Zu erledigen",
+    tip: "Tipp",
+    success: "Erfolg",
+    question: "Frage",
+    warning: "Warnung",
+    failure: "Misserfolg",
```

Comment:
Would go with 'Fehler' here - it's a little closer to error than failure, but sounds more natural and is used more.